scripts/irods/test/test_control_plane.py

```diff
@@ -82,6 +84,92 @@ def test_shutdown_local_server(self):
             finally:
                 IrodsController().start()
 
+    def test_shutdown_local_server_using_hostname_alias_defined_hosts_config__issue_5700(self):
+        try:
+            with lib.file_backed_up(paths.hosts_config_path()):
+                hostname_alias = 'hostname-alias.issue-5700'
+
+                with open(paths.hosts_config_path(), 'r+') as f:
+                    hosts_config = json.load(f)
+                    hosts_config['host_entries'] = [{
+                        'address_type': 'local',
+                        'addresses': [
+                            {'address': lib.get_hostname()},
+                            {'address': hostname_alias}
+                        ]
+                    }]
+
+                    f.seek(0)
+                    f.truncate(0)
+                    f.write(json.dumps(hosts_config, indent=4))
+
+                # Restart the server so that the origin server process sees the updates to
```

Comment:
origin -> original ?    or perhaps... initial?

Reply:
i used origin because i didn't want to use the word, grandpa.

i don't feel *original* and *initial* work very well. how about *root*?

Reply:
root server?  no, definitely already has a meaning on the internet / DNS.

hmmm...

Reply:
done.